### PR TITLE
feat(ui): flag trigger

### PR DESCRIPTION
**Pull request by bimalgrg519**
Fixes https://github.com/bucketeer-io/bucketeer/issues/314

### CODE REVIEW

File: ui/web-v2/apps/admin/src/assets/lang/en.json [hunk 1]
```diff
@@ -512,6 +515,30 @@
...
+  "trigger.deleteTriggerDialogMessage": "The trigger will be deleted permanently.",
+  "trigger.deleteTriggerDialogTitle": "Delete Trigger",
+  "trigger.description": "Use triggers to turn a flag on or off remotely. See the {link}",
+  "trigger.disableTrigger": "Disable Trigger",
+  "trigger.documentation": "documentation",
+  "trigger.editDescription": "Edit description",
+  "trigger.enableTrigger": "Enable Trigger",
+  "trigger.lastTriggered": "Last Triggered",
+  "trigger.resetTriggerDialogBtnLabel": "Reset",
+  "trigger.resetTriggerDialogMessage": "The current URL will become invalid. Ensure that you copy and save the new URL.",
```
Comment (cre8ivejp) [inline ui/web-v2/apps/admin/src/assets/lang/en.json:531]:
```suggestion
  "trigger.resetTriggerDialogMessage": "The current URL will become invalid. Ensure that you copy and store the new URL.",
```

File: ui/web-v2/apps/admin/src/assets/lang/en.json [hunk 2]
```diff
@@ -512,6 +515,30 @@
...
+  "trigger.resetTriggerDialogTitle": "Reset Trigger URL",
+  "trigger.resetURL": "Reset URL",
+  "trigger.triggerType": "Type",
+  "trigger.triggerURL": "Trigger URL",
+  "trigger.triggerUrlDescription": "Once you leave this page, the URL will be hidden.",
+  "trigger.triggerUrlTitle": "Copy and save this URL.",
```
Comment (cre8ivejp) [inline ui/web-v2/apps/admin/src/assets/lang/en.json:537]:
```suggestion
  "trigger.triggerUrlTitle": "Copy and store this URL.",
```

File: ui/web-v2/apps/admin/src/lang/messages.ts [hunk 3]
```diff
@@ -504,6 +504,106 @@ export const messages = {
...
+      id: 'trigger.resetURL',
+      defaultMessage: 'Reset URL',
+    }),
+    deleteTrigger: defineMessage({
+      id: 'trigger.deleteTrigger',
+      defaultMessage: 'Delete Trigger',
+    }),
+    triggerUrlTitle: defineMessage({
+      id: 'trigger.triggerUrlTitle',
+      defaultMessage: 'Copy and save this URL.',
```
Comment (cre8ivejp) [inline ui/web-v2/apps/admin/src/lang/messages.ts:571]:
```suggestion
      defaultMessage: 'Copy and store this URL.',
```

File: ui/web-v2/apps/admin/src/lang/messages.ts [hunk 4]
```diff
@@ -504,6 +504,106 @@ export const messages = {
...
+      defaultMessage: 'Delete',
+    }),
+    resetTriggerDialogTitle: defineMessage({
+      id: 'trigger.resetTriggerDialogTitle',
+      defaultMessage: 'Reset Trigger URL',
+    }),
+    resetTriggerDialogMessage: defineMessage({
+      id: 'trigger.resetTriggerDialogMessage',
+      defaultMessage:
+        'The current URL will become invalid. Ensure that you copy and save the new URL.',
```
Comment (cre8ivejp) [inline ui/web-v2/apps/admin/src/lang/messages.ts:596]:
```suggestion
        'The current URL will become invalid. Ensure that you copy and store the new URL.',
```

File: ui/web-v2/apps/admin/src/lang/messages.ts [hunk 5]
```diff
@@ -504,6 +504,106 @@ export const messages = {
...
   },
+  trigger: {
+    documentation: defineMessage({
+      id: 'trigger.documentation',
+      defaultMessage: 'documentation',
+    }),
+    description: defineMessage({
+      id: 'trigger.description',
+      defaultMessage:
+        'Use triggers to turn a flag on or off remotely. See the {link}',
```
Comment (cre8ivejp) [inline ui/web-v2/apps/admin/src/lang/messages.ts:515]: Please update the documentation link.
https://docs.bucketeer.io/feature-flags/creating-feature-flags/triggers

**Comment by kentakozuka** [comment]
@bimalgrg519 sorry can you run `/sign cla` again?

**Comment by kentakozuka** [comment]
Thanks 😄

**Comment by bimalgrg519** [comment]
Thank you @kentakozuka

**Review by cre8ivejp — APPROVED** [review]
Thank you!

**Pull request merged**